app/views/pages/aboutus.html.erb

```diff
@@ -1,4 +1,4 @@
-<strong><h2>About Us</h2></strong>
+<strong><h2><%= @page.name 'About Us' %></h2></strong>
```

Comment:
that seems very strange.  what is `@page.name` doing then?  perhaps we shouldn't have `@page.name` there then?